### DISCUSSION TITLE
R package and libicu... Is its-pointless on VACATIONS 😂😂

### DISCUSSION THREAD

**Discussion by dipayand** [termux/termux-packages#8202] [Q&A] [answered]
I am getting the following error on executing R 

CANNOT LINK EXECUTABLE "/data/data/com.termux/files/usr/lib/R/bin/exec/R": library "libicuuc.so.69" not found: needed by main executable

**Comment by ghost** [comment, marked as answer]
You'll need to use `R` in chroot, e.g. Debian or Arch. Repository by its-pointless is no longer maintained. We also don't have build recipes for `R` and can't provide a build that would be compatible with current `libicu`.

As stated in https://wiki.termux.com/wiki/Getting_started#Tips, do backups of the environment (`$PREFIX`) once everything is set up and working. Termux is rolling-release distribution and currently there no ways for us to provide different release models that would be more stable & friendly to third-party software on long term basis.

> Is its-pointless on VACATIONS

Could be, but probably retired.
